examples/nextjs-spa/src/pages/login.tsx

```diff
@@ -71,16 +76,16 @@ const Login: NextPageWithLayout = () => {
     }
 
     if (flowId) {
-      getLoginFlow(flowId).catch(
-        (err: AxiosError) =>
-          err.response?.status === 410 ?? createFlow(refresh, aal, returnTo),
-      ) // if for some reason the flow has expired, we need to get a new one
+      getFlow(flowId).catch(() => {
+        createFlow(refresh, aal, returnTo)
+      })
       return
     }
 
     // Otherwise we initialize it
     createFlow(refresh, aal, returnTo)
-  }, [createFlow, getLoginFlow, refresh, aal, returnTo, flowId, router.isReady])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.